lib/hanami/model/error.rb

```diff
@@ -88,5 +88,14 @@ def initialize(message = 'Check constraint has been violated')
         super
       end
     end
+
+    # Unknown database type error for repository auto-mapping
+    #
+    # @since x.x.x
+    class UnknownDatabaseTypeError < Error
+      def initialize(message)
```

Comment:
@jodosha actually, in rom-sql 1.0.0 inferrer will no longer raise that error, so that you can define missing attributes yourself. Can we expose schema definition in hanami repos? (in a similar way as assoc, I suppose). The reason for this change is flexibility, because we can't guarantee that inferrer will always work and that even if it works, the attributes are defined exactly like you want. Having a way of combining inference with explicit definitions is a win-win in such case, because you avoid boilerplate and still have complete control.